source/_data/plugins.yml

```diff
@@ -2816,12 +2816,18 @@
     - menu
     - categories
     - archive
+- name: Bridge
+  description: 'Admin panel for your website. Features: edit posts and pages, file manager, website settings editor, theme and plugin discovery.'
+  link: https://github.com/DeepSpaceHarbor/hexo-bridge
+  tags:
+    - admin
+    - editor
+    - admin-panel
 - name: hexo-next-pwa
   description: Yet, another pwa plugin
   link: https://github.com/jiangtj-lab/hexo-next-pwa
   tags:
     - PWA
     - Service Worker
     - Web App Manifest
-    - Workbox
-
+    - Workbox
```

Comment:
The blank line at the end of this file needs to be kept 😂

Reply:
4 attempts later, I did it. 😂 

Sorry about this,I thought yaml parsers are smart enough to find the end of file, even without the new line at the end.